ui/perfherder/helpers.js

```diff
@@ -446,6 +446,16 @@ export const getTextualSummary = (alerts, alertSummary, copySummary = null) => {
     const { suite, test, machine_platform: platform } = alert.series_signature;
     const extraOptions = alert.series_signature.extra_options.join(' ');
 
+    const taskLink =
+      'https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/';
+    const resultsPath =
+      '/runs/0/artifacts/public/test_info/browsertime-results.tgz';
+    const updatedAlert = alertSummary.alerts.find((a) => alert.id === a.id);
+    if (updatedAlert.task_id && updatedAlert.prev_task_id) {
+      const prevValueLink = `${taskLink}${updatedAlert.prev_task_id}${resultsPath}`;
```

Comment:
What if instead of using the `task_id`s provided by `StatusDropdown` to compute the links, we'd rather compute the links directly in `StatusDropdown` & make use of them here?

Reply:
Sounds like an interesting idea.